src/multiple_choice/config.py

```diff
@@ -51,8 +51,8 @@
 
 # default configurations
 # TODO: update version number before release
-default_conf_local = {'version': "2.7.1"}
-default_conf_syncd = {'version': "2.7.1"}
+default_conf_local = {'version': "2.7.2"}
```

Comment:
Incrementing the bugfix version is okay, if you would describe the current behaviour as unintended. I would rather have it as a feature version (`2.8.0`) but both is totally fine with me.